src/site/xdoc/server/config-blobstore.xml

```diff
@@ -53,14 +53,34 @@
                     The generated startup warning log can be deactivated via the <code>cassandra.blob.store.disable.startup.warning</code> environment
                     variable being positioned to <code>false</code>.
                 </dd>
+                Deduplication requires a garbage collector mechanism to effectively drop blobs. A first implementation
```

Comment:
Isn't this all block off the xml syntax and duplicated with the correct syntax below? (line 73)